src/app/pages/landing-page/landing-page.component.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
You forgot to add tests to cover app frame, app-product-list and app-product parts